orchestrate/obase/pipelines.go

```diff
@@ -6,8 +6,16 @@ import (
 	"github.com/relex/slog-agent/base"
 	"github.com/relex/slog-agent/base/bconfig"
 	"github.com/relex/slog-agent/base/bsupport"
+	"github.com/relex/slog-agent/util"
 )
 
+type pipelineWorkerSettings struct {
```

Comment:
may be better called like `outputBufferPairWorkerSet` or `outputWorkerSet`, something indicating output+buffer.